plugins/com.google.cloud.tools.eclipse.integration.appengine/src/com/google/cloud/tools/eclipse/integration/appengine/NewM2EclipseGeneratedProjectTest.java

```diff
@@ -47,14 +48,11 @@ public void testNewMavenAppEngineProject() throws CoreException {
 
     assertTrue(project.exists());
 
-    /*** Commented as it's not true at the moment. ***/
-    // IFacetedProject facetedProject = new FacetedProjectHelper().getFacetedProject(project);
-    // assertNotNull("Project should be faceted", facetedProject);
-    // assertTrue( new FacetedProjectHelper().projectHasFacet(facetedProject,
-    // AppEngineStandardFacet.ID));
-
     IFacetedProject facetedProject = new FacetedProjectHelper().getFacetedProject(project);
-    assertNull("Project should not be faceted", facetedProject);
+    assertNotNull("Project should be faceted", facetedProject);
```

Comment:
I know we will remove this test entirely, but it's really strange that the Travis build failed exactly at this point and that previously the opposite test was succeeding. I wonder if the test below isn't working somehow.